src/java/CollectorUrlSchemeUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ Copyright 2016 Esri
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.net.Uri;
+
+/**
+ * <p>
+ * Helper class to generate a Uri or to launch Collector for ArcGIS App.
+ * </p>
+ * Documented at <a href="https://github.com/ArcGIS/collector-integration#documentation">Collector integration</a>
+ */
+public final class CollectorUrlSchemeUtil {
+    private static final String COLLECTOR_URI = "com.esri.arcgis.collector";
+
+    private static final String ARCGIS_COLLECTOR_SCHEME = "arcgis-collector";
+
+    private static final String MAP_ITEM_ID = "itemID";
+
+    private static final String MAP_CENTER = "center";
+
+    private CollectorUrlSchemeUtil() {
+    }
```

Comment:
Since this is sample code, it might be good to explain that the constructor is private because this class is only intended to be used with its static methods.